editor/src/components/window/file.rs

```diff
@@ -182,22 +182,10 @@ impl FileImplementedEditor for super::EchidnaWindow {
     }
 
     fn action_save_file(&self) -> Result<(), Box<dyn Error>> {
-        match self.get_current_tab() {
-            Ok(page) => {
-                let page: EchidnaCoreEditor = page;
-                match page.file() {
-                    Some(file) => {
-                        let location = file.location();
-                        match page.save_file(None) {
-                            Ok(_) => Ok(()),
-                            Err(e) => Err(e),
-                        }
-                    }
-
-                    None => self.action_save_file_as(),
-                }
-            }
-            Err(e) => Err(e),
+        let page: EchidnaCoreEditor = self.get_current_tab()?;
+        match page.file() {
+            Some(_) => page.save_file(None),
```

Comment:
It seems I accidentally wrote `page.save_file(None)`. Can you change it to this?

```rs
page.save_file(file.location())
```

Reply:
Done https://github.com/null8626/Echidna/commit/1e492ba790917f4d7bc2de4feb5c456b306376e7